src/pages/MainPage/contactsSection.scss

```diff
@@ -0,0 +1,161 @@
+.contacts {
+    min-height: 41.81rem;
+
+    &-wrapper {
+        max-width: 67.81rem;
+        margin: 0 auto;
+        font-family: $fourth-font;
+    }
+
+    &-title,
+    &-subtitle,
+    &-info-title,
+    &-info-email {
+        margin: 0;
+        font-family: inherit;
+        font-weight: 400;
+    }
+
+    &-title {
+        font-size: 1.94rem;
+    }
+
+    &-subtitle {
+        font-size: 1.25rem;
+        text-align: center;
+    }
+
+    &-subtitle-form {
+        width: 25.25rem;
+    }
+
+    &-text {
+        margin: 1.44rem auto 0 auto;
+        font-family: $primary-font;
+        font-size: 1.38rem;
+    }
+
+    &-container {
+        display: grid;
+        grid-template-columns: 20.4rem 39.3rem;
+        gap: 4.33rem;
+        padding: 3.75rem 0 0 3.75rem;
+    }
+
+    &-info {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        padding-top: 1.2rem;
+
+        &-title {
+            margin-top: 0.75rem;
+            font-size: 1rem;
+        }
+
+        &-mail {
+            margin-top: 2.75rem;
+            border-radius: 0.625rem;
+            box-shadow: 6px 6px 6px 0px rgba($dark-color, 0.25);
+            width: 100%;
+            height: 8.19rem;
+            text-align: center;
+
+            img {
+                margin-top: 1.25rem;
+            }
+        }
+
+        &-email {
+            margin-top: 1.5rem;
+            font-size: 1.125rem;
+            text-decoration: none;
+            color: $dark-color;
+        }
+
+        &-links {
+            margin-top: 7.88rem;
+            width: 100%;
+            text-align: center;
+        }
+
+        &-list {
+            display: flex;
+            justify-content: space-between;
+            list-style: none;
+            margin: 1.19rem 0 0 0;
+            padding: 0;
+
+            &-item {
+                line-height: 2.25rem;
+            }
+        }
+    }
+
+    &-form {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+
+        h3 {
+            margin-bottom: 0.625rem;
+        }
+
+        form {
```

Comment:
it's not really a good practice to stick to tags in styles, it's better to create class names for that